build_tools/packaging/conda/build_audio.sh

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env bash
+if [[ -x "/remote/anaconda_token" ]]; then
+    . /remote/anaconda_token || true
+fi
+
+set -ex
+
+ # Function to retry functions that sometimes timeout or have flaky failures
+retry () {
+    $*  || (sleep 1 && $*) || (sleep 2 && $*) || (sleep 4 && $*) || (sleep 8 && $*)
+}
+
+if [ "$#" -ne 1 ]; then
```

Comment:
because torchaudio doesn't have any CUDA C++ files, you can simply delete this ` if` condition and the usage of `desired_cuda` i guess?